pennylane/_qubit_device.py

```diff
@@ -673,6 +691,15 @@ def vn_entropy(self, wires, log_base):  # pragma: no cover
         """
         raise NotImplementedError
 
+    def mutual_info(self, wires0, wires1, log_base):  # pragma: no cover
+        """Returns the mutual information prior to measurement.
+
+        .. note::
+
+            Only simulators support this property.
```

Comment:
There exist hardware simulators too. For now, such simulators would not support `mutual_info` either.
```suggestion
            Only simulators that are capable of returning the state support this property.
```